fic/eri/v1/router_paired_to_gcp_connections/requests.go

```diff
@@ -81,17 +92,19 @@ type Destination struct {
 
 // Source represents source parameter for connection.
 type Source struct {
-	RouterID    string      `json:"routerId" required:"true"`
-	GroupName   string      `json:"groupName" required:"true"`
-	RouteFilter RouteFilter `json:"routeFilter" required:"true"`
+	RouterID    string       `json:"routerId" required:"true"`
+	GroupName   string       `json:"groupName" required:"true"`
+	RouteFilter RouteFilter  `json:"routeFilter" required:"true"`
+	Primary     SourceHAInfo `json:"primary" required:"true"`
+	Secondary   SourceHAInfo `json:"secondary" required:"true"`
 }
 
 // CreateOpts represents options used to create a connection.
 type CreateOpts struct {
 	Name        string      `json:"name" required:"true"`
 	Source      Source      `json:"source"`
 	Destination Destination `json:"destination"`
```

Comment:
Can you make Destination filed required?

Reply:
DONE